sn_networking/src/record_store.rs

```diff
@@ -48,13 +50,27 @@ use tokio::sync::mpsc;
 use walkdir::{DirEntry, WalkDir};
 use xor_name::XorName;
 
-// A spend record is at the size of 4KB roughly.
-// Given chunk record is maxed at size of 512KB.
-// During Beta phase, it's almost one spend per chunk,
-// which makes the average record size is around 256k.
-// Given we are targeting node size to be 32GB,
-// this shall allow around 128K records.
-const MAX_RECORDS_COUNT: usize = 128 * 1024;
+/// The default value of targted max storage space is to be 32GB.
+const DEFAULT_MAX_STORAGE_SPACE: u64 = 32 * 1024 * 1024 * 1024;
+
+lazy_static! {
+    /// The max storage space for the records.
+    /// A `node size` is to be defined as this plus the logging space assigned.
+    pub static ref MAX_STORAGE_SPACE: u64 = std::option_env!("MAX_STORAGE_SPACE")
+        .unwrap_or("34359738368")
```

Comment:
Just to avoid trying to calculate what this number might be

```suggestion
        .unwrap_or(DEFAULT_MAX_STORAGE_SPACE.to_string())
```